src/js/templates/logo.ts

```diff
@@ -1,4 +1,4 @@
-export default (position, hide) => {
+export default (position: string, hide: boolean): string => {
```

Comment:
I added an HTMLTemplateString type so we can better identify template usage, and add in actual string template detection when it becomes a feature. We can take it out if it's not needed, but if you like it we can use it here for return type (and a few others)